slick.core.js

```diff
@@ -38,6 +48,8 @@
      */
     this.stopImmediatePropagation = function () {
       isImmediatePropagationStopped = true;
+      if(nativeEvent)
+        nativeEvent.stopImmediatePropagation();
```

Comment:
could you wrap this `if` and the other function `if` as well in curly brace to avoid potential issues in the future

Reply:
Done